recipe/tests/check.c

```diff
@@ -0,0 +1,174 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
@ocefpaf, can we use these files from the source tarball?

Reply:
Sure. I'll try that.